aws_tools/dynamodb_handler.py

```diff
@@ -77,48 +77,53 @@ def delete_item(self, keys):
                 if isinstance(value, dict) and 'condition' in value and 'value' in value:
-                    condition = value['condition']
+                    condition_str = value['condition']
                     value = value['value']
-                    if 'value2' in value:
-                        value2 = value['value2']
+                    if condition_str == "between":
+                        value, value2 = value["value"], value["value2"]
                 else:
-                    condition = 'eq'
+                    value = None
+                    condition_str = 'eq'
 
                 if not value and only_fields_with_values:
                     continue
 
-                if condition == 'eq':
-                    filter_expression &= Attr(field).eq(value)
-                if condition == 'ne':
-                    filter_expression &= Attr(field).ne(value)
-                if condition == 'lt':
-                    filter_expression &= Attr(field).lt(value)
-                if condition == 'lte':
-                    filter_expression &= Attr(field).lte(value)
-                if condition == 'gt':
-                    filter_expression &= Attr(field).gt(value)
-                if condition == 'gte':
-                    filter_expression &= Attr(field).gte(value)
-                if condition == 'begins_with':
-                    filter_expression &= Attr(field).begins_with(value)
-                if condition == 'between':
-                    filter_expression &= Attr(field).between(value, value2)
-                if condition == 'ne':
-                    filter_expression &= Attr(field).ne(value)
-                if condition == 'is_in':
-                    filter_expression &= Attr(field).is_in(value)
-                if condition == 'contains':
-                    filter_expression &= Attr(field).contains(value)
+                if condition_str == 'eq':
+                    condition = Attr(field).eq(value)
+                elif condition_str == 'ne':
+                    condition = Attr(field).ne(value)
+                elif condition_str == 'lt':
+                    condition = Attr(field).lt(value)
+                elif condition_str == 'lte':
+                    condition = Attr(field).lte(value)
+                elif condition_str == 'gt':
+                    condition = Attr(field).gt(value)
+                elif condition_str == 'gte':
+                    condition = Attr(field).gte(value)
+                elif condition_str == 'begins_with':
+                    condition = Attr(field).begins_with(value)
+                elif condition_str == 'between':
+                    condition = Attr(field).between(value, value2)
+                elif condition_str == 'ne':
+                    condition = Attr(field).ne(value)
+                elif condition_str == 'is_in':
+                    condition = Attr(field).is_in(value)
+                elif condition_str == 'contains':
```

Comment:
This needs to be an `elif`. If it was just `if`, then any `condition` other than `"contains"` (even valids ones such as `"eq"`) would cause an expection to be thrown.